docs/mutation-data-transcript-annotation.md

```diff
@@ -0,0 +1,85 @@
+clinical sequencing at MSKCC. The `uniprot` set of transcripts was constructed several years ago, but we are no longer 
+certain about the logic on how to reconstruct them hence they are not being kept up to date. One can see the 
+differences between the two in [this file](https://github.com/cBioPortal/cbioportal-
+frontend/files/9498680/genes_with_different_uniprot_mskcc_isoforms.txt). For the public cBioPortal (https:
+//www.cbioportal.org) we are using `mskcc`, for for the GENIE cBioPortal (https://genie.cbioportal.org) we use `uniprot`
+. Although for local installations the default currently is `uniprot` to avoid people upgrading from a previous version 
+of cBioPortal to accidentally switch to `mskcc`, we do recommend to use `mskcc`. 
+
+#### How default transcript assignment affects the Mutations Tab 
+
+The [Mutations Tab](https://bit.ly/39hVtDd) shows the full protein sequence. The one shown by default is the canonical 
+transcript (`mskcc` or `uniprot` depending on configuration). The mutations are drawn on the lollipop based on the 
+protein position found in the cBioPortal database. For the [public cBioPortal](https://cbioportal.org) all mutation 
+data in MAF format are annotated using [Genome Nexus](https://www.genomenexus.org) to add the gene and protein change 
+columns. This is then imported into the cBioPortal database. Whether you choose to use the set of `uniprot` or `mskcc` 
+transcripts, make sure to indicate it in the [Genome Nexus Annotation Pipeline](https://github.com/genome-nexus/genome-
+nexus-annotation-pipeline#maf-annotation)(`--isoform-override <mskcc or uniprot>`) when annotating as well as in the 
+properties file of cBioPortal. That way the [Mutations Tab](https://bit.ly/39hVtDd) will show the correct canonical 
+transcript. Note that whenever somebody uses the dropdown on the Mutations Tab to change the displayed transcript, 
+Genome Neuxs re-annotates all mutations on the fly. The browser sends over the genomic location (chrom,start,end,ref,
+alt) to get the protein change information for each transcript. 
+
+#### Plans for default transcripts 
+
+We are planning to move to a single set of default transcripts over time, previously `uniprot` was used for the public 
+facing portals and is currently still the default for new installations. Our plan is to use `mskcc` everywhere and 
+eventually we will most likely move to [MANE](https://www.ensembl.org/info/genome/genebuild/mane.html). MANE is only 
+available for grch38 and since most of our data is for grch37 this is currently not feasible. Whichever set of 
+transcripts you choose to use, make sure to indicate so in the [Genome Nexus Annotation Pipeline](https:
+//github.com/genome-nexus/genome-nexus-annotation-pipeline#maf-annotation) (`--isoform-override <mskcc or uniprot>`) 
+and put the same set of transcripts in the properties file of cBioPortal, such that the [Mutations Tab](https:
```

Comment:
a link to the relevant properties line or the doc page that mentions this property would be nice!

Reply:
added